### DISCUSSION TITLE
Accessibility Statement

### DISCUSSION THREAD

**Discussion by richardwmanley** [SafeExamBrowser/seb-win-refactoring#535] [General]
Hi all, I can see there is work ongoing to improve accessibility with WCAG 2.1 AA as the target.  We would like to adopt SEB but in the UK have an obligation to review accessibility in detail.  Is there a published Accessibility Statement and VPAT self-assessment?

**Comment by dbuechel** [comment]
Thanks for your interest in the new accessibility functionalities. We're indeed aiming to release such a document by January / February 2023.

**Comment by richardwmanley** [comment]
Dear Damian,

Thank you for getting back to me.

To be clear, please can you confirm if you have published any Accessibility Statement already?

Regards,

Richard.

—
Richard Manley | Strategic Business Partner
IT Services, University of the West of England, Coldharbour Lane, Bristol  BS16 1QY

Tel: 0117 32 83811
Chat with me in Microsoft Teams<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchat%2F0%2F0%3Fusers%3DRichard.Manley%40uwe.ac.uk&data=04%7C01%7CRichard.Manley%40uwe.ac.uk%7Cf77e2d9dfd0e4c32188508d9d05f1fcd%7C07ef1208413c4b5e9cdd64ef305754f0%7C0%7C0%7C637769929228934441%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=eZrnbp9SQM%2BVQGPsuzzXdepGhXAw8obwPlrR9RtjClQ%3D&reserved=0>


From: Damian Büchel ***@***.***>
Date: Friday, 9 December, 2022 at 18:09
To: SafeExamBrowser/seb-win-refactoring ***@***.***>
Cc: Richard Manley ***@***.***>, Author ***@***.***>
Subject: Re: [SafeExamBrowser/seb-win-refactoring] Accessibility Statement (Discussion #535)

Thanks for your interest in the new accessibility functionalities. We're indeed aiming to release such a document by January / February 2023.

**Reply by dbuechel** [reply]
No, at the moment we do not yet have any accessibility statement or declaration.

**Comment by christianp** [comment]
Has any progress been made on this? I can help with advice on the general structure of an accessibility statement, if that would help.

**Reply by dbuechel** [reply]
Yes indeed, we actually have the VPAT finished for Windows, but due to vacation we're waiting to complete the VPAT for macOS / iOS to be then able to publish them all together on our website.

**Reply by christianp** [reply]
Fab, thanks!
